backend/app/poller.py

```
@@ -142,46 +142,33 @@ async def send_push_notifications(notifications, device_tokens, loop):
-            android_config = messaging.AndroidConfig(priority='high')
-            
-            messages.append(messaging.Message(
-                notification=messaging.Notification(title=content['title'], body=content['body']),
-                token=token,
-                android=android_config,
-                data=data_payload if data_payload else None
-            ))
+            subscription_info = json.loads(json.dumps(token))
+            messages.append((subscription_info, json.dumps(data_payload)))
 
     if not messages: return
```

Ruff (0.15.20)

[error] Multiple statements on one line (colon)

(E701)